tests/validators/allow_empty_string_test.py

```diff
@@ -67,13 +67,14 @@ def test_invalid_not_empty_string_value():
 
     @staticmethod
     def test_invalid_type_contains_empty_string():
-        """ Test that AllowEmptyString adds '' to the expected_types parameter if the wrapped validator raises an InvalidTypeError. """
+        """ Test that AllowEmptyString adds str to the expected_types parameter if the wrapped validator raises an InvalidTypeError. """
         validator = AllowEmptyString(DecimalValidator())
         with pytest.raises(ValidationError) as exception_info:
             validator.validate(123)
+        print(exception_info.value.to_dict())
```

Comment:
There shouldn't be `print` statements in the test code, please remove the line. :)

Reply:
Fixed